demo/articles/arabic.html

```diff
@@ -52,7 +52,7 @@
             </p>
         </div>
         <!-- Scripts -->
-        <script src="../wikipedia-preview.development.js"></script>
```

Comment:
we are not able to use this in `npm run dev`, this is only available when we run `npm run build`

the solution can be script target="module" to import it to use; then when we deploy to demo site, we need to replace it to normal script (either umd or iife format)